<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateCustomFormatDialog</class>
 <widget class="QDialog" name="CreateCustomFormatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Custom Format</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <item>
    <widget class="QLineEdit" name="FormatName">
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>Format Name</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="audioFormatLabel">
       <property name="text">
        <string>Audio Format</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="vidoeFormat"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="videoFormatLabel">
       <property name="text">
        <string>Video Format</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextBrowser" name="audioFormatInfo"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="audioFormat"/>
     </item>
     <item row="3" column="0">
      <widget class="QTextBrowser" name="videoFormatInfo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
